vulnerabilities/tests/test_api.py

```diff
@@ -229,6 +235,18 @@ def test_api_with_single_vulnerability(self):
             ],
             "affected_packages": [],
             "references": [],
+            "weaknesses": [
+                {
+                    "cwe_id": 119,
+                    "name": "Improper Restriction of Operations within the Bounds of a Memory Buffer",
+                    "description": "The software performs operations on a memory buffer, but it can read from or write to a memory location that is outside of the intended boundary of the buffer.",
+                },
+                {
+                    "cwe_id": 10000,
+                    "name": "",
+                    "description": "",
+                },
```

Comment:
@TG1999 to remove this part should I edit the `WeaknessSerializer` ( add a validation function ) or what should I do ?